HostApp/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const ModuleFederationPlugin = require("webpack/lib/container/ModuleFederationPlugin");
+
+const deps = require("./package.json").dependencies;
+module.exports = (_, argv) => ({
+  output: {
+    publicPath: "http://localhost:9000/",
+  },
+
+  resolve: {
+    extensions: [".tsx", ".ts", ".jsx", ".js", ".json"],
+  },
+
+  devServer: {
+    port: 9000,
+    historyApiFallback: true,
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.m?js/,
+        type: "javascript/auto",
+        resolve: {
+          fullySpecified: false,
+        },
+      },
+      {
+        test: /\.(css|s[ac]ss)$/i,
+        use: ["style-loader", "css-loader", "postcss-loader"],
+      },
+      {
+        test: /\.(ts|tsx|js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+        },
+      },
+    ],
+  },
+
+  plugins: [
+    new ModuleFederationPlugin({
+      name: "HostApp",
+      filename: "remoteEntry.js",
+      remotes: {
+        landingApp: 'landingApp@http://localhost:9001/remoteEntry.js',
```

Comment:
I would suggest to not include landingApp, since that code will live in the host. As well change client for personal and enterprise for company

Reply:
added inside of HostApp